src/components/cocktail.js

```diff
@@ -52,13 +52,11 @@ export default class Cocktail extends React.Component {
               </ol>
             </div>
           </div>
-          <input onChange={this.updateQ} className="mt-10 mb-2 border-b-2 border-black" type="text" placeholder="City / State" ref={this.loc}/>
-          <a 
-            href={`/search/yelp?q=${name}&loc=${this.state.loc}`}
-            className="flex flex-col w-60 font-montserrat font-semibold border-2 border-black px-8 py-2 text-xl "
-          >
-            Find near you <span className="text-xs italic">Powered by Yelp</span>
-          </a>
+          <form method="get" action={`/search/yelp?q=${name}&loc=${this.state.loc}`} >
```

Comment:
I'll pin this form for later but it breaks the React frame work. It's easier this way but it's not something you would see in a React app. Sorta the downside of React. It flys in the face of reason in parts like this.